documentation/API-Testing-Guidelines.md

```diff
@@ -125,27 +125,27 @@ For the Feature description, API name and version must be included. When the fea
 Feature: CAMARA Device location verification API, v0.2.0 - Operation verifyLocation
 ```
 
-### Environment variables <a name="environment-variables"></a>
+### Environment variables
 
-Commonly, some values to fill the request bodies will not be known in advance and cannot be specified as part of the feature file, as they will be specific for the test environment, and they will have to be provided by the implementation to the tester as a separate set of environment or configuration variables.
+Commonly, some values to fill the request bodies will not be known in advance and cannot be specified as part of the feature file, as they will be specific for the test environment, and they will have to be provided by the implementation of the tester as a separate set of environment or configuration variables.
```

Comment:
```suggestion
Commonly, some values to fill the request bodies will not be known in advance and cannot be specified as part of the feature file, as they will be specific for the test environment, and they will have to be provided by the implementation to the tester as a separate set of environment or configuration variables.
```

Reply:
It is "to", because it is something to be provided to the person/entity that performs the testing